src/libcore/num/f64.rs

```diff
@@ -237,7 +237,9 @@ impl Float for f64 {
     /// Converts to degrees, assuming the number is in radians.
     #[inline]
     fn to_degrees(self) -> f64 {
-        self * (180.0f64 / consts::PI)
+        // Use a constant for better precision.
+        let pis_in_180 = 57.2957795130823208767981548141051703_f64;
```

Comment:
Shouldn't there be more digits here than in `f32`?

Reply:
Both literals actually have more digits than necessary: they both have 36 significant figures, which was chosen to be consistent with the existing definitions of mathematical constants (for both `f32` and `f64`).